functions/Copy-DbaResourceGovernor.ps1

```diff
@@ -109,26 +113,93 @@ function Copy-DbaResourceGovernor {
             Notes             = $null
             DateTime          = [DbaDateTime](Get-Date)
         }
+        
+        $copyResourceGovClassifierFunc = [pscustomobject]@{
+            SourceServer      = $sourceServer.Name
+            DestinationServer = $destServer.Name
+            Type              = "Resource Governor Settings"
+            Name              = "Classifier Function"
+            Status            = $null
+            Notes             = $null
+            DateTime          = [DbaDateTime](Get-Date)
+        }
 
         if ($Pscmdlet.ShouldProcess($destination, "Updating Resource Governor settings")) {
             if ($destServer.Edition -notmatch 'Enterprise' -and $destServer.Edition -notmatch 'Datacenter' -and $destServer.Edition -notmatch 'Developer') {
                 Write-Message -Level Verbose -Message "The resource governor is not available in this edition of SQL Server. You can manipulate resource governor metadata but you will not be able to apply resource governor configuration. Only Enterprise edition of SQL Server supports resource governor."
             }
             else {
                 try {
-                    $sql = $sourceServer.ResourceGovernor.Script() | Out-String
-                    Write-Message -Level Debug -Message $sql
-                    Write-Message -Level Verbose -Message "Updating Resource Governor settings."
-                    $destServer.Query($sql)
+                    Write-Message -Level Verbose -Message "Managing classifier function."
+                    if (!$sourceClassifierFunction) {
```

Comment:
Do you have a potential of ever having more than one classifier function?

If so this should be adjusted to handle multiple being found.

Reply:
The resource governor can have only one classifier function bound.